corehq/apps/cloudcare/static/cloudcare/js/formplayer/hq.events.js

```diff
@@ -16,14 +16,8 @@ FormplayerFrontend.module("HQ.Events", function(Events, FormplayerFrontend) {
         // For Chrome, the origin property is in the event.originalEvent object
         var origin = event.origin || event.originalEvent.origin,
             data = event.data,
-            returnValue = null,
```

Comment:
this was unfortunate, but it's impossible to use postMessage with callbacks since it only accepts serializable data. the only other option is to use `eval` which i don't want to do. i think we'll have to use this as a one way messaging API